README.md

```diff
@@ -1,19 +1,19 @@
 # practices
 
-This document intends to document how we develop software at 2DII, including
+This document intends to document how we develop software at RMI, including
 social and technical aspects around a number of themes. It should help guide
 our actions,  and understand what to expect from our colleagues and what they
 expect from us.  Wherever possible, we refer to the source of information
 that defines a "best practice". Anecdotes and personal experience have
 value, but we prefer evidence from research, books, and large and mature
 communities of practice.
 
-We have agreed to this practices by consensus. They document heuristics, as
+We have agreed to these practices by consensus. They document heuristics, as
```

Comment:
did we? LOL

Reply:
I mean, in principle, we agreed on these 3 years ago hahahah. Definitely up for interpretation if we still agree to this, but that's exactly why I'm bringing attention back to this repo :-)